modelon/impact/client/jupyterhub/authorize.py

```diff
@@ -37,23 +35,24 @@ def authorize(
     )
     jupyter_context = sal.JupyterContext(base=context)
     service = service or sal.JupyterHubService()
+    jupyter_token = jupyter_context.token
 
     try:
-        jupyter_context.token = _get_jupyter_token(credential_manager, interactive)
+        jupyter_token = _get_jupyter_token(credential_manager, interactive)
```

Comment:
We still need to set the token for the jupyter_context, assigning the variable would have to be after we have done this.